ecosystem-tool-user-guide.md

```diff
@@ -1,272 +1,86 @@
 ---
-title: TiDB Ecosystem Tools User Guide
+title: TiDB Ecosystem Tools Overview
 category: reference
 aliases: ['/docs/dev/reference/tools/user-guide/','/docs/dev/how-to/migrate/from-mysql/','/docs/dev/how-to/migrate/incrementally-from-mysql/','/docs/dev/how-to/migrate/overview/']
 ---
 
-# TiDB Ecosystem Tools User Guide
+# TiDB Ecosystem Tools Overview
 
-The TiDB ecosystem has a wealth of tools for data migration, backup & restore for users with different use cases to choose from. 
+This document introduces the functionalities of TiDB ecosystem tools and their relationship.
 
-- Some of the functionalities of these tools might overlap. For example, TiDB Loader, TiDB Lightning and TiDB DM can all do full data loading. 
-- Some of the tools might have evolved. For example, TiDB Binlog will be evolved to CDC (Change Data Capture). 
-- Some of the tools are designed to support specific TiDB versions and the others might be deprecated as user requirements change.
+## Full data export 
 
-This guide is specifically designed to help you better understand these tools and therefore make an informed decision while choosing these tools to support your business.
+[Dumpling](/export-or-backup-using-dumpling.md) is a tool for the logical full data export from MySQL or TiDB.
 
-## Data import (restore or data replication)
+The following are the basics of Dumpling:
 
-### Full data import tools
+- Input: MySQL/TiDB cluster
+- Output: SQL/CSV file
+- Supported TiDB versions: all versions
+- Kubernetes support: No
 
-#### TiDB Lightning
+## Full data import
 
-[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) (Lightning) is a tool used for the fast full import of large amounts of data into a TiDB cluster. Currently, TiDB Lightning supports reading SQL dump exported via Mydumper or CSV data source.
+[TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) (Lightning) is a tool used for the full import of large amounts of data into a TiDB cluster. Currently, TiDB Lightning supports reading SQL dump exported via Mydumper or CSV data source.
```

Comment:
It's better to use `Dumpling` to replace `Mydumper` now.

Reply:
Updated.